resources/views/statistics/index.blade.php

```diff
@@ -0,0 +1,46 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="jumbotron jumbotron-fluid text-center">
+        <h1>Statistics</h1>
+        <p class="lead">Here you can find statistics about elephpants.</p>
+    </div>
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-md-10">
+                <div class="mb-3">
+                </div>
+                <div class="card">
+                    <div class="card-header">
+                        <strong>Ownership of elephpants</strong>
+                    </div>
+                    <div class="card-body">
+                        <table class="table table-striped table-bordered mb-0">
+                            <thead>
+                            <tr>
+                                <th scope="col" width="7%" class="text-center">#</th>
+                                <th scope="col" width="35%">Name</th>
+                                <th scope="col" width="12%" class="text-center">Ownership percent</th>
+                                <th scope="col" width="12%" class="text-center">Number</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            @foreach($elephpants as $key => $elephpant)
+                                <tr>
+                                    <td class="text-center align-middle">{{ $key + 1 }}</td>
+                                    <td>
+                                        <span class="font-weight-bold">#{{ $elephpant->id }} - {{ $elephpant->name }}</span><br/>
+                                        {{ $elephpant->description }}
+                                    </td>
+                                    <td class="text-center align-middle">{{ round((($elephpant->nbElephpant/$nbUsers) * 100), 2) }}%</td>
+                                    <td class="text-center align-middle">{{ $elephpant->nbElephpant }}</td>
```

Comment:
here you are not considering the quantity of elephpants each user has correct? for the % calculation is correct but I think here it would be good to show the total number of elephpants. ideas?

Reply:
Yes, it's a good idea.

I added a new column with the total number of elephpants per species.

Reply:
![image](https://user-images.githubusercontent.com/1866496/73536731-e0d93a00-4426-11ea-96d2-66bb4a1d6681.png)

But I'm not really sure about labels on columns. Ideas?

Reply:
about labels I'd say `#, Name, Ownership, Users, Total`, but this is up to you! let me know if you want to change this and then I merge. thank you!

Reply:
Sounds perfect! I changed it 😄